### PR TITLE
test: omnibar tree-extension tests [ET-203]

### PR DESCRIPTION
**Pull request by EmilyBonar**
## Ticket

[ET-203]

## Description

Adding tests to the tree-extension portion of the omnibar to boost code coverage.

## Checklist

- [ ] Changes have been manually QA'd
- [ ] New features have been approved by the corresponding PM
- [ ] User-facing API changes have the "User-facing API Change" label
- [ ] Release notes have been added as a separate file under `docs/release-notes/`
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses have been included for new code which was copied and/or modified from any external code

[ET-202]: https://hpe-aiatscale.atlassian.net/browse/ET-202?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DET-123]: https://hpe-aiatscale.atlassian.net/browse/DET-123?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[ET-203]: https://hpe-aiatscale.atlassian.net/browse/ET-203?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: webui/react/src/omnibar/tree-extension/index.test.tsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+describe('absPathToAddress', () => {
+  it('Absolute Path', () => {
+    const path = generateTreePath();
+    expect(absPathToAddress(path)).toEqual(['1', '2', '3', '4']);
+  });
+});
+
+describe('queryTree', () => {
+  it('Leaf Node child', async () => {
```
Comment (JComins000) [inline webui/react/src/omnibar/tree-extension/index.test.tsx:46]: write it like a sentence
it "should ..."

File: webui/react/src/omnibar/tree-extension/index.test.tsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+describe('parseInput', () => {
+  it('Parsing input', async () => {
+    expect(await parseInput('node', NON_LEAF_NODE())).toEqual({
+      path: [NON_LEAF_NODE()],
+      query: 'node',
+    });
+  });
+});
+
+describe('absPathToAddress', () => {
```
Comment (JComins000) [inline webui/react/src/omnibar/tree-extension/index.test.tsx:38]: this looks more like it's only testing your helper method

**Review by keita-determined — APPROVED** [review]
web change lgtm. leave a test file to @JComins000 for review

File: webui/react/src/omnibar/tree-extension/index.test.tsx [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+const LEAF_NODE = (title?: string): LeafNode => ({
+  onAction: handleAction,
+  title: title ?? 'leaf node',
+});
+const NON_LEAF_NODE = (title?: string): NonLeafNode => ({
+  options: [LEAF_NODE()],
+  title: title ?? 'non-leaf node',
+});
+
+const generateTreePath = (length: number = 5) => {
```
[the remark this answers is not in the document]
Comment (EmilyBonar) [reply webui/react/src/omnibar/tree-extension/index.test.tsx:21]: Changed it from random to alternating

**Review by JComins000 — APPROVED** [review]

**Pull request merged**